docs/reference/ml/trained-models/apis/get-trained-models-stats.asciidoc

```diff
@@ -97,6 +97,16 @@ The detailed allocation status given the deployment configuration.
 (integer)
 The current number of nodes where the model is allocated.
 
+`error_count`:::
+(integer)
+The total number of errors when evaluating the trained model
+for all nodes in the deployment.
+
+`rejected_execution_count`:::
+(integer)
+The total number of inference requests that were not processed because
```

Comment:
> The total number .. queue was full for all nodes

This sentence gives me pause because it makes me wonder whether it's the "total number of requests for all nodes" or "the queue was full on all nodes".  If it's the former, perhaps it's clearer if we split the sentence in two?: 

> The number of inference requests that were not processed because the queue was full. It is the total value for all nodes in the deployment.

Reply:
This figure is the deployment level sum of the `rejected_execution_count` for each node in the deployment. Further down each node has timeout, rejected execution and error counts, this value is the high level total.

I've had another go in   https://github.com/elastic/elasticsearch/pull/82705/commits/eb2ed6d65b02e24117174e809c6bf5e5f45a55cb